controllers/orderController.ts

```diff
@@ -0,0 +1,164 @@
+import { Request, Response } from "express";
+import db from "../models";
+import { error } from "console";
+
+// This method returns all users
+const getAllOrders = async (req: Request, res: Response) => {
+  try {
+    const orders = await db.Order.findAll({
+      attributes: {
+        exclude: ["updatedAt"],
+      },
+      include: [
+        {
+          model: db.Product,
+          as: "products",
+          attributes: ["id", "quantity"],
+        },
+        {
+          model: db.Address,
+          as: "address",
+        },
+      ],
+    });
+
+    if (!orders.length) {
+      res.status(404).json({ error: "No orders found" });
+      return;
+    }
+
+    res.json(
```

Comment:
You don't need a loop, I think the SQL query returns the required results without looping